app/views/users/_account.html.erb

```diff
@@ -121,30 +121,17 @@
 </h4>
 <br>
 <script>
-  var deleteAccountBtn = document.getElementById('delete__account__btn')
-  var deleteAccountUsernameInput = document.getElementById('delete__account__username__field')
-  var deleteAcccountVerificationInput = document.getElementById('delete__account__verification__field')
+  var deleteAccountBtn = document.getElementById('delete__account__btn');
```

Comment:
If you’re in here cleaning up, might as well convert `var`s to `const`s.

Reply:
Have taken care of, @nickytonline. Thanks :)